.github/PULL_REQUEST_TEMPLATE/icon_addition.md

```diff
@@ -16,8 +16,6 @@ App Name (`com.package.app` → `drawable.svg`)
 App Name (`com.package.app`)
 App Name (`com.package.app`)
 
-## Contributor's checklist
+## Contributor's check
```

Comment:
honestly i think it was fine how it was before

Reply:
I explained the issues with the current solution in the description and changelog.

Besides, there is no useful reason why you need to put 3 checkboxes with keyboard. One is enough.

Reply:
If it's just about the header, I decided to change that because the word "lists" itself indicates that there will be 2+ items.

Reply:
i think "checklist" makes more sense though since we're still using the bullet point.

Reply:
Clarified the definition of "checklist". Returned the old version, because the checkbox description implies several actions.